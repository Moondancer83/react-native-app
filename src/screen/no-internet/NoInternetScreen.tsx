import React from 'react';
import {Text, View} from 'react-native';

import {style} from './NoInternetSreen.style';

export default function NoInternetScreen() {
  return (
    <View style={style.view}>
      <Text style={style.title}>No internet connection</Text>
      <Text style={style.body}>
        Application will load when device is connected to the internet.
      </Text>
    </View>
  );
}
