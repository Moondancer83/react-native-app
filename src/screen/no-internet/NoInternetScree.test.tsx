import React from 'react';
import renderer from 'react-test-renderer';

import NoInternetScreen from './NoInternetScreen';

test('Test NoInternetScreen against snapshot', () => {
  const component = renderer.create(<NoInternetScreen />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
