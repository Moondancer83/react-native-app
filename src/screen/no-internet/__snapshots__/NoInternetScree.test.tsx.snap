// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test NoInternetScreen against snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
      "padding": 10,
    }
  }
>
  <Text
    style={
      Object {
        "fontFamily": "Avenir Next",
        "fontSize": 28,
        "fontWeight": "600",
        "marginBottom": 20,
        "marginTop": -50,
      }
    }
  >
    No internet connection
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "Avenir Next",
        "fontSize": 15,
        "fontWeight": "500",
      }
    }
  >
    Application will load when device is connected to the internet.
  </Text>
</View>
`;
